hack/update-codecgen.sh

```diff
@@ -52,7 +98,9 @@ for generated_file in ${generated_files}; do
 	rm -f "${generated_file}"
 done
 
-for generated_file in ${generated_files}; do
+# Generate files in the dependency order.
+for current in ${index[@]}; do
```

Comment:
Also, you want `"${index[@]}"` (because bash and expansion and quotes and sigh).